### PR TITLE
support extract output from armTemplate deployment

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: lib/testScenario/postmanCollectionRunnerClient.ts [hunk 1]
```diff
@@ -371,6 +371,7 @@ export class PostmanCollectionRunnerClient implements TestScenarioRunnerClient {
       raw: JSON.stringify(body, null, 2),
     });
     this.addAuthorizationHeader(item);
+    console.log(armTemplate.outputs);
```
Comment (raych1) [inline lib/testScenario/postmanCollectionRunnerClient.ts]: Will output include sensitive data?

File: lib/testScenario/postmanCollectionRunnerClient.ts [hunk 2]
```diff
@@ -341,13 +343,11 @@ export class PostmanCollectionRunnerClient implements TestScenarioRunnerClient {
     this.auth(stepEnv.env);
     const item = new Item();
     item.name = step.step;
-    const path =
-      "/subscriptions/:subscriptionId/resourcegroups/:resourceGroupName/providers/Microsoft.Resources/deployments/{{deploymentName}}?api-version=2020-06-01";
```
Comment (raych1) [inline lib/testScenario/postmanCollectionRunnerClient.ts:345]: where does `deploymentName` come from before?

File: lib/testScenario/postmanCollectionRunnerClient.ts (hunk 1 above)
Comment (ruowan) [reply lib/testScenario/postmanCollectionRunnerClient.ts]: Updated. It's just a debug log. I remove it.

File: lib/testScenario/postmanCollectionRunnerClient.ts (hunk 2 above)
Comment (ruowan) [reply lib/testScenario/postmanCollectionRunnerClient.ts:345]: From env. but env is not define. So the value is undefine. It's a bug before.

**Review by zhenglaizhang — APPROVED** [review]

**Review by raych1 — APPROVED** [review]

**Pull request merged**